### PR TITLE
Removes uninitialized variable in function __CFMessagePortCreateRemote in CFMessagePort

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by valeriyvan** [comment]
@spevans could you be so kind to review this PR?

**Comment by millenomi** [comment]
I'm going to aggressively move PRs to drafts if they have unaddressed changes. Feel free to move this back to an actual PR when it's ready again.

File: CoreFoundation/RunLoop.subproj/CFMessagePort.c [hunk 1]
```diff
@@ -443,8 +443,7 @@ static CFMessagePortRef __CFMessagePortCreateRemote(CFAllocatorRef allocator, CF
     CFMachPortContext ctx;
     uint8_t *utfname = NULL;
     CFIndex size;
-    mach_port_t port;
-    kern_return_t ret;
+    mach_port_t port = 0;
```
[the remark this answers is not in the document]
Comment (valeriyvan) [reply CoreFoundation/RunLoop.subproj/CFMessagePort.c]: Fixed

**Comment by millenomi** [comment]
@swift-ci please test and merge

**Pull request merged**